src/ui/components/Icon/styles.scss

```diff
@@ -2,7 +2,7 @@
 @import "~core/css/inc/mixins";
 
 @mixin icon($name) {
-  background: url("./#{$name}.svg") center no-repeat;
+  background: url("./img/icons/#{$name}.svg") center no-repeat;
```

Comment:
I think you should put everything in `./img`, not `./img/icons/`. I really can't foresee any case where we'd need an extra subdirectory.